From gadiyar@ti.com  Wed Feb 17 15:58:21 2010
From: Anand Gadiyar <gadiyar@ti.com>
Date: Fri, 12 Feb 2010 17:48:59 +0530
Subject: USB: ehci: omap: use default interrupt threshold
Cc: Anand Gadiyar <gadiyar@ti.com>
Message-ID: <1265977140-9701-2-git-send-email-gadiyar@ti.com>


The current driver reduces the interrupt threshold to 1 microframe.
This was an accidental change and is not really required.
The default of 8 microframes will do just fine. So change it back.

Signed-off-by: Anand Gadiyar <gadiyar@ti.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ehci-omap.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/usb/host/ehci-omap.c
+++ b/drivers/usb/host/ehci-omap.c
@@ -651,10 +651,6 @@ static int ehci_hcd_omap_probe(struct pl
 	/* cache this readonly data; minimize chip reads */
 	omap->ehci->hcs_params = readl(&omap->ehci->caps->hcs_params);
 
-	/* SET 1 micro-frame Interrupt interval */
-	writel(readl(&omap->ehci->regs->command) | (1 << 16),
-			&omap->ehci->regs->command);
-
 	ret = usb_add_hcd(hcd, irq, IRQF_DISABLED | IRQF_SHARED);
 	if (ret) {
 		dev_dbg(&pdev->dev, "failed to add hcd with err %d\n", ret);
